src/ripple/unity/app_ledger.cpp

```diff
@@ -34,11 +34,12 @@
 #include <ripple/app/ledger/OrderBookIterator.cpp>
 #include <ripple/app/ledger/TransactionStateSF.cpp>
 
+#include <ripple/app/ledger/impl/ConsensusImp.cpp>
 #include <ripple/app/ledger/impl/DisputedTx.cpp>
 #include <ripple/app/ledger/impl/InboundLedger.cpp>
 #include <ripple/app/ledger/impl/InboundLedgers.cpp>
 #include <ripple/app/ledger/impl/LedgerCleaner.cpp>
-#include <ripple/app/ledger/impl/LedgerConsensus.cpp>
+#include <ripple/app/ledger/impl/LedgerConsensusImp.cpp>
```

Comment:
A change like that seems like a good idea.  I found myself getting a bit dizzy distinguishing between a `Consensus` and a `LedgerConsensus` in NetworkOPs.  `ConsensusRound` might be a nice name, since there are a lot of kinds of `Round` in the world.